feyre_backend/backend_service/app.py

```diff
@@ -0,0 +1,14 @@
+from flask import Flask
+from backend_service.api.controller.roll_controller import roll_api
+from backend_service.api.controller.stats_controller import stats_api
+
+def create_app():
+    app = Flask(__name__)
+    app.register_blueprint(roll_api, url_prefix='/api/backendService/roll')
+    app.register_blueprint(stats_api, url_prefix='/api/backendService/stats')
```

Comment:
```suggestion
    app.register_blueprint(roll_api, url_prefix='/api/backendservice/roll')
    app.register_blueprint(stats_api, url_prefix='/api/backendservice/stats')
```

URIs should always be in all lowercase

Reply:
Also, URIs should be treated the same way whether it has a trailing Slash or not. This will be important for Kong.

For example:

```
/api/backendservice/roll
/api/backendservice/roll/
```

Should resolve to the same endpoint. I believe in the current state, a trailing slash `/` will result in a 404.